Firmware/RTK_Everywhere/GNSS.ino

```diff
@@ -130,6 +160,11 @@ bool gnssSurveyInReset()
             // Put UM980 into rover mode to cancel base averaging mode
             return (um980SetModeRoverSurvey());
         }
+        else if (present.gnss_mosaicX5)
+        {
+            // Put mosaicX5 into rover mode to cancel base averaging mode
+            return (mosaicX5SetModeRoverSurvey());
```

Comment:
The function to put the mosaic into Rover mode is ModeRoverSurvey()? Why the Survey. I'll keep reading...

Reply:
Would ModeSurveyReset be more fitting?

Reply:
```mosaicX5SetModeRoverSurvey``` is the X5 version of ```gnssSurveyInReset```
```gnssSurveyInReset``` is only called during the state transition from ```STATE_BASE_TEMP_SURVEY_STARTED``` to ```STATE_ROVER_NOT_STARTED```. Comments say "Survey-In took more than %d minutes. Returning to rover mode"
Yeah. Agreed. The name needs to change. Let's go with ```mosaicX5SurveyReset```.